### DISCUSSION TITLE
Error with port undefined

### DISCUSSION THREAD

**Discussion by Arountasief** [aws/aws-sdk-js#4100] [Q&A]
Hello,

I have got an error in my project after my credentials have been updated:

`Inaccessible host: *** at port undefined`

I think the problem comes from a change in my endpoint, I now have a port specified.
Old endpoint : #.com 
New endpoint : #.com:*****

And I can access to my data with the new credentials if I use a terminal.

Does anyone have an idea where the problem could come from ?

**Comment by ajredniwja** [comment]
Hey @Arountasief thanks for opening this issue, apologies could not get to the issue earlier, is it still a persisting issue? If it is can you please share the code that you are using to make these calls and also mention how you set up the credentials? Thanks
